extensions/mssql/src/connection/commands.ts

```diff
@@ -0,0 +1,23 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AppContext } from '../appContext';
+import * as vscode from 'vscode';
+import * as constants from './constants';
+import { TelemetryReporter } from '../telemetry';
+import { ConnectionService } from './connectionService';
+
+const clearPooledConnectionsAction = 'clearPooledConnections';
+
+export function registerConnectionCommands(appContext: AppContext) {
+	appContext.extensionContext.subscriptions.push(vscode.commands.registerCommand('mssql.clearPooledConnections', async () => {
+		TelemetryReporter.sendTelemetryEvent(clearPooledConnectionsAction);
```

Comment:
there's already telemetry sent when commands are run (can see these in the commands tab in the ADS power bi report), so this isn't necessary if no other information is getting sent

Reply:
Done